pkg/workflow/workload/pod/builder.go

```diff
@@ -185,6 +185,26 @@ func (m *Builder) CreateVolumes() error {
 		})
 	}
 
+	// Add common volume to pod if configured
+	i := 1
+	for _, v := range m.wfr.Spec.PresetVolumes {
```

Comment:
I'd prefer `i, v := range ...`, it's ok to skip some of the index when met PV type.